valhalla/odin/enhancedtrippath.h

```diff
@@ -557,7 +557,7 @@ class EnhancedTripLeg_Node {
     return mutable_node_->type();
   }
 
-  uint32_t elapsed_time() const {
+  double elapsed_time() const {
     return mutable_node_->elapsed_time();
```

Comment:
technically didnt need to do this as its only for debug output but, it didnt feel right to truncate the number